site/content/en/docs/manual/advanced/formats/format-yolov8-classification.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 'YOLOv8-Classification'
+linkTitle: 'YOLOv8-Classification'
+weight: 7
+description: 'How to export and import data in YOLOv8 Classification format'
+---
+
+For more information, see:
+
+- [Format specification](https://docs.ultralytics.com/datasets/classify/)
+- [Dataset examples](https://github.com/cvat-ai/datumaro/tree/develop/tests/assets/yolo_dataset/yolov8_classification)
+
+## YOLOv8 Classification export
+
+For export of images:
+
+- Supported annotations: Tags.
+- Attributes: Not supported.
+- Tracks: Not supported.
+
+The downloaded file is a .zip archive with the following structure:
+
+```bash
+archive.zip/
+├── train
+│    ├── labels.json  # optional cvat extension. Contains original ids and labels
```

Comment:
From what I understand this file is not optional and required when importing annotations. If this is the case, I think clarifications is needed.

Reply:
Rewritten the comment